docs/DafnyRef/UserGuide.md

```diff
@@ -287,16 +291,16 @@ then runs it with the three command-line arguments `1 2 3`
 #### 25.5.1.7. `dafny server` {#sec-dafny-server}
 
 The `dafny server` command starts the Dafny Language Server, which as an [LSP-compliant](https://microsoft.github.io/language-server-protocol/) implementation of Dafny.
-Note two important points:
-- The [Dafny VSCode extension]() uses this LSP implementation
-- This LSP implementation is designed so that its behavior (other than IDE specific aspects) matches that of the command-line `dafny` tool
+The [Dafny VSCode extension]() uses this LSP implementation, which in turn uses the same core Dafny implementation as the command-line tool.
 
 The Dafny Language Server is described in more detail [here](#sec-dafny-language-server-vscode).
 
 #### 25.5.1.8. `dafny audit` {#sec-dafny-audit}
```

Comment:
It seems strange to mention a command that's not merged yet, but OK.

Reply:
I wouldn't normally do so. If the audit is not merged by the next release, I'll remove this section.